R/token-define.R

```diff
@@ -1,31 +1,34 @@
-token <- tibble::tribble(
-  ~text,           ~class,         ~token,
-  "&",          "logical",          "AND",
-  "&&",         "logical",         "AND2",
-  "|",          "logical",           "OR",
-  "||",         "logical",          "OR2",
-  ">",          "logical",           "GT",
-  "<",          "logical",           "LT",
-  "<=",         "logical",           "LE",
-  ">=",         "logical",           "GE",
-  "!=",         "logical",           "NE",
-  "==",         "logical",           "EQ",
-  "=",      "assign_left",       "EQ_SUB",
-  "=",      "assign_left",    "EQ_ASSIGN",
-  "<-",     "assign_left",  "LEFT_ASSIGN",
-  "->",    "assign_right", "RIGHT_ASSIGN",
-  "+",             "math",          "'+'",
-  "-",             "math",          "'-'",
-  "*",             "math",          "'*'",
-  "/",             "math",          "'/'",
-  "^",             "math",          "'^'",
-  "~",          "formula",          "'~'",
-  "if",            "cond",           "IF",
-  "else",          "cond",         "ELSE",
-  "in",       "loop_cond",           "IN",
-  "while",    "loop_cond",        "WHILE"
+# styler: off
```

Comment:
@lorenzwalthert This directive is tripping up `precommit`. 

Can I remove it? It will get rid of the neat alignment here, though.

Reply:
I don’t think we should remove it without further ado. Maybe open another issue? This should work out of the box.

Reply:
Sounds good. I will create an issue.

Reply:
Created issue to track this: https://github.com/r-lib/styler/issues/1019